cypher/cypher-docs/src/docs/dev/ql/merge/index.asciidoc

```diff
@@ -50,10 +50,6 @@ If there are multiple matches, they will all be passed on to later stages of the
 The last part of `MERGE` is the `ON CREATE` and `ON MATCH`.
 These allow a query to express additional changes to the properties of a node or relationship, depending on if the element was `MATCH` -ed in the database or if it was `CREATE` -ed.
 
-The rule planner (see <<how-are-queries-executed>>) expands a `MERGE` pattern from the end point that has the variable with the lowest lexicographical order.
```

Comment:
Since it is possible to use the RULE planner by using Cypher 3.1 planner=rule, should we really remove all rule planner documentation? Or do we rely on uses that use 'Cypher 3.1' to also refer to 'Neo4j 3.1' documentation in for situations like this? I guess that would be sensible, because we would not want to include all possible version content in each set of documentation.

Reply:
This is a good question, and I think one for the Docs Team. @mariascharin @jjaderberg @davidoliverSP2 please feel free to reinstate this text / add links / resolve how you see best.

Reply:
👍